account/compat.py

```diff
@@ -0,0 +1,13 @@
+import django
+
+try:
+    from django.core.urlresolvers import resolve, reverse, NoReverseMatch
+except ImportError:
+    from django.urls import resolve, reverse, NoReverseMatch  # noqa
+
+
+def is_authenticated(user):
+    if django.VERSION >= (1, 10):
+        return user.is_authenticated
```

Comment:
LOL. I bet this change in Django is going to magically fix people's code and instantly make their sites more secure ;-)